test/function_test.py

```diff
@@ -13,16 +13,16 @@
 False
 >>> is_same_data(s,[])
 False
->>> filter_color("yellow")
-'yellow'
->>> filter_color((0.2,0.3,0.4))
-(0.2, 0.3, 0.4)
->>> filter_color("#FFFFFF")
-'#FFFFFF'
+>>> filter_color("yellow", "blue")
+('yellow', 'blue')
+>>> filter_color((0.2,0.3,0.4),(0.2,0.3,0.4,1))
+((0.2, 0.3, 0.4), (0.2, 0.3, 0.4, 1))
+>>> filter_color("#FFFFFF","#ffffe1")
+('#FFFFFF', '#ffffe1')
 >>> random.seed(2)
->>> color1 = filter_color("random")
+>>> color1, bgcolor1 = filter_color("random","random")
 >>> random.seed(3)
->>> color2 = filter_color("RANDOM")
+>>> color2,bgcolor2 = filter_color("RANDOM","RANDOM")
```

Comment:
`color2, bgcolor2`

Reply:
fixed in https://github.com/sepandhaghighi/samila/pull/131/commits/8371989046bc9748c389876c91d8a2049fdce471